app/components/autocomplete/slash_suggestion/slash_suggestion.js

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {
+    FlatList
+} from 'react-native';
+
+import {RequestStatus} from 'mattermost-redux/constants';
+
+import {makeStyleSheetFromTheme} from 'app/utils/theme';
+
+import SlashSuggestionItem from './slash_suggestion_item';
+
+const SLASH_REGEX = /(^\/)([a-zA-Z-]*)$/;
+const FIVE_MINS = 1000 * 60 * 5;
```

Comment:
Can we make this descriptive somehow like `SOMETHING_SOMETHING_TTL_MS` so that it's easier to grok at a glance?

Reply:
Will do.